app/domain/sub09/SubscriptionDisplayResponse.scala

```diff
@@ -96,6 +104,7 @@ object SubscriptionResponse {
   implicit val euVatFormat = Json.format[EUVATNumber]
   implicit val xiSubscriptionFormat = Json.format[XiSubscription]
   implicit val returnParametersFormat = Json.format[ReturnParameters]
+  implicit val VATIDsFormat = Json.format[vatIDs]
```

Comment:
better to rename it to vatIDFormat as val should follow the lowerCamelCase

Reply:
Renamed it to vatIDFormat